ignite/contrib/handlers/lr_finder.py

```diff
@@ -241,6 +244,16 @@ def plot(
             lrs = lrs[skip_start:-skip_end]
             losses = losses[skip_start:-skip_end]
 
+        # Check to show the suggested learning rate
+        if display_suggestion:
+            idx = self._min_grad_idx + skip_start - skip_end
+            sug_lr = lrs[idx]
+            equivalent_loss = losses[idx]
```

Comment:
We can also either find the loss from the history which corresponds to suggested lr or simply draw vertical line.